specifyweb/frontend/js_src/lib/components/WorkBench/WbToolkit.tsx

```diff
@@ -0,0 +1,206 @@
+      role="toolbar"
+    >
+      {hasPermission('/workbench/dataset', 'transfer') &&
+      hasTablePermission('SpecifyUser', 'read') ? (
+        <>
+          <Button.Small
+            aria-haspopup="dialog"
+            aria-pressed={toolkitOptions.changeOwner.show}
+            className="wb-change-data-set-owner"
+            onClick={toolkitOptions.changeOwner.open}
+            disabled={hasUnSavedChanges}
+            title={hasUnSavedChanges ? wbText.unavailableWhileEditing() : ""}
+          >
+            {wbText.changeOwner()}
+          </Button.Small>
+          <Button.Small
+            aria-haspopup="dialog"
+            aria-pressed={toolkitOptions.devPlan.show}
+            className="wb-show-plan"
+            onClick={toolkitOptions.devPlan.open}
+          >
+            {wbText.uploadPlan()}
+          </Button.Small>
+        </>
+      ) : undefined}
+      <Button.Small
+        className="wb-export-data-set"
+        onClick={handleExport}
+        disabled={hasUnSavedChanges}
+        title={hasUnSavedChanges ? wbText.unavailableWhileEditing() : ""}
+      >
+        {commonText.export()}
+      </Button.Small>
+      <span className="-ml-1 flex-1" />
+      {hasPermission('/workbench/dataset', 'update') && (
+        <>
+          <Button.Small
+            aria-haspopup="dialog"
+            aria-pressed={toolkitOptions.coordinatesConversion.show}
+            className="wb-convert-coordinates"
+            title={wbText.unavailableWithoutLocality()}
+            onClick={toolkitOptions.coordinatesConversion.open}
+            disabled={!hasLocality}
+          >
+            {wbText.convertCoordinates()}
+          </Button.Small>
+          <Button.Small
+            aria-haspopup="dialog"
+            aria-pressed={toolkitOptions.geoLocate.show}
+            className="wb-geolocate"
+            title={wbText.unavailableWithoutLocality()}
+            onClick={toolkitOptions.geoLocate.open}
+            disabled={!hasLocality}
+          >
+            {localityText.geoLocate()}
+          </Button.Small>
+        </>
+      )}
+      <Button.Small
+        aria-haspopup="dialog"
+        aria-pressed={toolkitOptions.leafletMap.show}
+        className="wb-leafletmap"
+        title={wbText.unavailableWithoutLocality()}
+        onClick={toolkitOptions.leafletMap.open}
+        disabled={!hasLocality}
+      >
+        {localityText.geoMap()}
+      </Button.Small>
+      {toolkitOptions.changeOwner.show && (
+        <ChangeOwner
+          dataset={dataset}
+          onClose={toolkitOptions.changeOwner.close}
+        />
+      )}
+      {toolkitOptions.devPlan.show && (
+        <DevShowPlan
+          dataSetId={dataset.id}
+          dataSetName={dataset.name}
+          uploadPlan={dataset.uploadplan ?? ({} as UploadPlan)}
+          onChanged={(plan) => {
+            overwriteReadOnly(dataset, 'uploadplan', plan);
+            // TODO: figure out this trigger
+            // trigger('refresh');
```

Comment:
I believe that would do a browser refresh? I think the original trigger was reloading the dataset, which is what I'm trying to do with this function: 
https://github.com/specify/specify7/blob/2f4bc62caa273daebd4b8a7bf5a9dfa83514a566/specifyweb/frontend/js_src/lib/components/WorkBench/index.tsx#L63-L65

Reply:
good point - that would be better UX then entire page reload.
I forgot that this might be called often, unlike the other places, where page reload is used very sparingly